controllers/greptimedbcluster/greptimedbcluster_controller.go

```diff
@@ -220,6 +220,29 @@ func (r *Reconciler) validate(ctx context.Context, cluster *v1alpha1.GreptimeDBC
 		return fmt.Errorf("no components spec in cluster")
 	}
 
+	if cluster.Spec.Frontend != nil && cluster.Spec.Frontend.TLS != nil {
+		if len(cluster.Spec.Frontend.TLS.CA.Name) > 0 {
+			err := r.Get(ctx, types.NamespacedName{Namespace: cluster.Namespace, Name: cluster.Spec.Frontend.TLS.CA.Name}, &corev1.Secret{})
+			if err != nil {
+				return fmt.Errorf("get CA secret '%s' failed, error: '%v'", cluster.Spec.Frontend.TLS.CA.Name, err)
+			}
+		}
+
+		if len(cluster.Spec.Frontend.TLS.Cert.Name) > 0 {
+			err := r.Get(ctx, types.NamespacedName{Namespace: cluster.Namespace, Name: cluster.Spec.Frontend.TLS.Cert.Name}, &corev1.Secret{})
+			if err != nil {
+				return fmt.Errorf("get cert secret '%s' failed, error: '%v'", cluster.Spec.Frontend.TLS.Cert.Name, err)
```

Comment:
The three secrets name is same in `config/samples/frontend-tls/cluster.yaml`

Reply:
Here will get three same name secret

Reply:
Maybe the design of the tls field in `GreptimeDBCluster` is a little complicated, let me refactor that.

Reply:
@daviderli614 I already refactor the code to use `secretName` instead of secretKeySelector, it will make it easy to use.

Reply:
It look good to me